osi_lane.proto

```diff
@@ -192,69 +192,91 @@ message Lane
-        // \note The center line is the line that a typical vehicle follows more
-        // or less (depending on the situation, a little more to the left or
-        // right of the center line). The intended direction of travel on the
-        // lane is given by the direction defined by the sequence of points
-        // forming the \c #centerline w.r.t. \c
-        // #centerline_is_driving_direction.
+        repeated Vector3d surfaceline = 3;
+        
+        // The ID of the surface line.
+        // Example: sl4 (see reference picture HighwayExit).
         //
-        // \note The \c #centerline is defined only for \c #type =
-        // \c #TYPE_DRIVING and if exactly one or no \c #lane_pairing pair
-        // exists.
+        // \note Note ID is global unique.
         //
-        // \note Intersections and non-driving lanes do not have a center line.
-        // A vehicle must calculate this individually and depending on the
-        // situation.
+        // \rules
+        // is_globally_unique
+        // \endrules
         //
-        repeated Vector3d centerline = 3;
+        optional Identifier surfaceline_id = 4;
 
-        // Definition of the intended driving direction.
+        // Definition of the placement of the newly introduced surfaceline in the center of the lane.
         //
         // Defined and used for driving lanes.
-        // \c true means driving direction is according to ascending storage
-        // order of center line points. \c false means driving direction is
-        // according to descending storage order of center line points.
+        // \c true means that the surface line is also the centerline of the lane4.
         //
-        // Example: \c #centerline_is_driving_direction = \c true for lane l4
-        // and \c #centerline_is_driving_direction = \c false for lane l2 in
+        // Example: \c #surfaceline_is_centerline = \c true for surfaceline sl2
+        // and \c #surfaceline_is_centerline = \c false for surfacelane sl1 in
         // image \ref HighwayExit .
         //
-        // \note The \c #centerline_is_driving_direction is defined for \c #type
+        // \note The \c #centerline is currently only defined for \c #type
         // = \c #TYPE_DRIVING .
         //
-        optional bool centerline_is_driving_direction = 4;
+        optional bool surfaceline_is_centerline = 5;
+        
+        // List of IDs of all surfaceline segments that are directly adjacent to the
+        // surfaceline on the left side (w.r.t. ascending order of surfaceline points
+        // and lane boundary points). Note that lengths of surfaceline segments are 
+        // not synchronized and therefore can contain multiple adjacent segments.
+        //
+        // \note If none or just one surfaceline is present, this may be empty.
+        //
+        // \note Up for discussion: If surface line is adjacent to a left lane boundary,
+        // this could contain the left_lane_boundary_id
```

Comment:
I would say it should be left empty in that case, because the information would just be duplicated. The field `repeated Identifier left_lane_boundary_id = 9;` should be sufficient IMO.